gtoolkit/run.sh

```diff
@@ -108,8 +108,9 @@ gtoolkit::prepare_gt() {
   local download_name
   local target
   gtoolkit_image_url="$(gtoolkit::archive_url)"
+  smalltalk_name="$(basename "${gtoolkit_image_url}" .zip)"
```

Comment:
`smalltalk_name` is the name of the selected image, e.g., `GToolkit64-alpha`. Rather introduce a new variable to handle this difference.

Reply:
I pushed a new commit that does as you suggest